github-markdown.css

```diff
@@ -677,6 +757,15 @@
   border-color: #0366d6;
 }
 
+.markdown-body .new-label-color-input:invalid {
+  color: #cb2431;
+}
+
+.markdown-body .tab-size[data-tab-size="8"] {
+  -moz-tab-size: 8;
+  tab-size: 8;
+}
```

Comment:
Can you exclude this one?

Reply:
This one is needed for rendering the code snippet.  In fact, I'm worried rules for other tab sizes are missing...  Will dig deeper.

Reply:
The defaults are really bad. We should change it to `tab-size: 4;`, which is what most would want.

Reply:
I think we should leave this stylesheet as compatible with GitHub's default styles as possible.  Clients can always override these styles if desired.  I tweaked the generator to include all possible tab size rules, though I'm not sure what would cause GitHub to output HTML that uses them.